gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/mfa_challenge_alternatives.html

```diff
@@ -41,7 +41,7 @@
                     </div>
                 </div>
 
-                <div th:if="${factor.factorType == 'SMS' || factor.factorType == 'CALL'}" class="radio-card">
+                <div th:if="${factor.factorType == 'SMS'}" class="radio-card">
```

Comment:
There are another set of mfa_enroll and mfa_alternative files in gravitee-am-managemment-api-rest/**/resources folder. Those files are for rendering themes. 
You need to update those two files as well. Those files are duplicate, so you can copy and paste.